public/pages/DetectorDetail/containers/DetectorDetail.tsx

```diff
@@ -197,20 +202,21 @@ export const DetectorDetail = (props: DetectorDetailProps) => {
     scroll(0, 0);
   }, []);
 
-  // Getting all visible indices. Will re-fetch if changes to the detector (e.g.,
+  // Getting all visible indices & aliases. Will re-fetch if changes to the detector (e.g.,
   // detector starts, result index recreated or user switches tabs to re-fetch detector)
   useEffect(() => {
-    const getInitialIndices = async () => {
+    const getInitialIndicesAliases = async () => {
       try {
         await dispatch(getIndices('', dataSourceId));
+        await dispatch(getAliases('', dataSourceId));
       } catch (error) {
         console.error(error);
         core.notifications.toasts.addDanger('Error getting all indices');
```

Comment:
nit: Error getting all indices and aliases?

Reply:
yes, changed to core.notifications.toasts.addDanger('Error getting all indices or aliases');